setup.sh

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env bash
+set -eu
+
+mkdir -p $INSTALL_DIR
+git clone https://gerrit.wikimedia.org/r/mediawiki/core $INSTALL_DIR/mediawiki
+git clone https://gerrit.wikimedia.org/r/mediawiki/skins/Vector $INSTALL_DIR/mediawiki/skins/Vector
+cd $INSTALL_DIR/mediawiki
+docker run -it --rm --user $(id -u):$(id -g) -v ~/.composer:/tmp -v $(pwd):/app docker.io/composer install
```

Comment:
Is this an alias for https://hub.docker.com/_/composer/? If so, I don't mind using the full url rather than the implicit short name they advertise. But, I ask because I've never seen the full url variant, nor am I able to verify that they are indeed the same (authoritatively). I mostly know `docker.io` as the Debian alternative to the `docker-ce` package.

Reply:
This came from the README, and I hadn't seen it before either. Like you say, I also believe it's an alias for `_/composer`. If the canonical example in the README changes it to use `composer` then I think we could do the same here.

Reply:
Okay, let's leave it this way then. Thanks!